utility/src/main/java/org/oppia/android/util/parser/CustomBulletSpan.kt

```diff
@@ -21,10 +21,15 @@ class CustomBulletSpan(context: Context) : LeadingMarginSpan {
   private val bulletRadius: Int
   private val gapWidth: Int
   private val yOffset: Int
+  /** The space between the parent and the bullet. */
   private val spacingBeforeBullet: Int
+  /** The space between the bullet and the text. */
   private val spacingBeforeText: Int
+  /** The total spacing between the bullet and the text.
```

Comment:
KDoc formatting should either be:

```kotlin
/** Something on one line. */
```

or

```kotlin
/**
 * Something on multiple lines, usually because it won't fit in 100 characters, or
 * because there are multiple lines of information to convey.
 */
```

Notice specifically where the slashes & asterisks go, and how they're aligned.

Please fix this case to match. :)

Reply:
I've removed this multiline as suggested in the previous comment